docker/image-builder.sh

```diff
@@ -7,18 +7,48 @@
 script_location=`realpath -s $(dirname ${BASH_SOURCE[0]})`
 android_cuttlefish_root_dir=$(realpath -s $script_location/..)
 
-if [[ "$1" == "" ]]; then
-    tag=cuttlefish-orchestration
-else
-    tag=$1
-fi
+usage() {
+  echo "usage: $0 [-t <tag>] [-b <build-option>]"
```

Comment:
I suggest to make this script defaults to building the regular image, using the published packages. Then if we pass flag `-d` meant for `local development` we then use the `packages` built locally.

Let's avoid the use of `apt-install` as an option name as `apt install` is used both ways.

Reply:
Done.